spi/imx6ull-qspi/qspi.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX 6ULL QSPI lib API
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Hubert Badocha
+ *
+ * %LICENSE%
+ */
+
+#ifndef IMX6ULL_QSPI_API_H
+#define IMX6ULL_QSPI_API_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <sys/threads.h>
+#include <phoenix/arch/imx6ull.h>
+
+
+#define MAX_READ_LEN  0xFFFF
+#define MAX_WRITE_LEN 0xFFFF
+
+typedef enum {
+	lut_stop,
+	lut_cmd,
+	lut_addr,
+	lut_dummy,
+	lut_mode,
+	lut_mode2,
+	lut_mode4,
+	lut_read,
+	lut_write,
+	lut_jmp_on_cs,
+	lut_addr_ddr,
+	lut_mode_ddr,
+	lut_mode2_ddr,
+	lut_mode4_ddr,
+	lut_read_ddr,
+	lut_write_ddr,
+	lut_data_learn
+} lut_code_t;
+
+#define LUT_INSTR(code, pad, operand) ((((code)&0x3F) << 10) | (((pad)&0x03) << 8) | ((operand)&0xFF))
+
+enum {
+	lut_pad1,
+	lut_pad2,
+	lut_pad4,
+};
+
+typedef struct {
+	uint16_t instrs[8];
+} lut_seq_t;
+
+typedef enum {
```

Comment:
Same in other places, two blank lines
```suggestion
} lut_seq_t;


typedef enum {
```